ProjectSettings/EditorSettings.asset

```diff
@@ -3,19 +3,24 @@
 --- !u!159 &1
 EditorSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 7
-  m_ExternalVersionControlSupport: Visible Meta Files
+  serializedVersion: 8
+  m_ExternalVersionControlSupport: Hidden Meta Files
```

Comment:
This is actually a problem. Needs to be changed back, otherwise when we move files it can cause  object references to break.

Edit > Project Settings > Editor
then
Version Control / Mode: “Visible Meta Files”

Reply:
Dunno how this got changed; updated.